cypress/integration/chi-custom-elements/text-input.spec.js

```diff
@@ -0,0 +1,59 @@
+const SELECTORS = {
+  chi_text_input: 'chi-text-input',
+  chi_helper_message: 'chi-helper-message',
+  chi_icon: 'chi-icon',
+  label: '.chi-label'
+};
+const CLASSES = {
+  warning: '-warning',
+  danger: '-danger',
+  success: '-success'
+};
+
+describe('Text Input', () => {
+  before(() => {
+    cy.visit('tests/custom-elements/text-input.html');
+  });
+
+  describe('Helper Message', () => {
+    beforeEach(() => {
+      cy.get('.test-helper-message-md')
```

Comment:
Can we create a variable for this class and reuse it?

```suggestion
      cy.get(HELPER_MESSAGE_MD)
```